documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-pass-through-database-driver-configuration-properties.adoc

```diff
@@ -0,0 +1,5 @@
+The {prodname} connector provides for pass-through configuration of the database driver.
+Pass-through database properties begin with the prefix `database.*`.
+For example, the connector passes properties such as `database.foobar=false` to the JDBC URL.
+
+As is the case with the xref:{context}-pass-through-database-history-properties-for-configuring-producer-and-consumer-clients[pass-through properties for database history clients], {prodname} strips the prefixes from the properties before it passes them to configure the database history producer or consumer client.
```

Comment:
```suggestion
As is the case with the xref:{context}-pass-through-database-history-properties-for-configuring-producer-and-consumer-clients[pass-through properties for database history clients], {prodname} strips the prefixes from the properties before it passes them to the database driver.
```

!!!

Reply:
Good catch. Neglected to update the sentence after I moved it from the other section. Thanks!